tests/test.rs

```diff
@@ -1,11 +1,14 @@
 #[macro_use]
 extern crate structure;
```

Comment:
Is this still needed with the new edition? It should be `use structure::structure;`.

Reply:
Fixed in 6da4216. Will squash these commits into one, before the PR is merged.